qemu/tests/balloon_service.py

```diff
@@ -0,0 +1,220 @@
+import logging
+import time
+import re
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import error_context
+from avocado.core import exceptions
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Balloon service test for windows guest.
+    1) boot a windows guest with balloon device.
+    2) enable and check driver verifier in guest.
+    3) install balloon service in guest.
+    4) enable polling in qmp.
+    5) evict and enlarge balloon.
+    6) get polling value in qmp.
+    7) uninstall balloon service and clear driver verifier.
+    """
+
+    def get_disk_vol(session):
+        """
+        Get virtio-win disk volume letter.
+
+        :param session: VM session.
+
+        :return drive_letter: virtio-win disk volume letter.
+        """
+        volume_name = params["cdrom_virtiowin"]
+        volume_name = volume_name.split("/")[-1]
+        volume_name = volume_name.split(".iso")[0]
```

Comment:
strange(".iso"),
volume_name = volume_name[:volume_name.rfind(".")] will be better.